db/migrate/20110601153519_really_regenerate_runtime_dependencies_list.rb

```diff
@@ -19,7 +19,7 @@ def self.each_dependency
       inner join rubygems on rubygems.id = dependencies.rubygem_id
       where scope = 'development'
     SQL
-    connection.select_all(dependencies).each_with_index do |row, index|
+    connection.select_all(dependencies).each_with_index do |row, _|
```

Comment:
Wait…why not just use `#each` here instead of `#each_with_index` if we’re not actually using the index?